src/txn/txn_timestamp.c

```diff
@@ -790,16 +790,18 @@ __wt_txn_set_prepare_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t prepare_
 int
 __wt_txn_set_read_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t read_ts)
 {
+    WT_DECL_RET;
     WT_TXN *txn;
     WT_TXN_GLOBAL *txn_global;
     WT_TXN_SHARED *txn_shared;
     wt_timestamp_t ts_oldest;
     char ts_string[2][WT_TS_INT_STRING_SIZE];
-    bool did_roundup_to_oldest;
+    bool did_roundup_to_oldest, using_pinned_ts;
```

Comment:
Minor nit, ignore if you want: For some reason `use` seems much clearer than `using` to me and I'm not sure why.